### ISSUE TITLE
Style guide and PEP8 conformance?

### ISSUE THREAD

**Issue by guitarmanvt** [oasis-open/cti-stix-elevator#2]
Which Python style guide is this project following? Are there any particular PEP-8 exceptions that are being allowed? (Extra credit: Which pylint or flake8 codes are ignored?)

I know this is a proof-of-concept, and as such it might not need to have an answer to the above questions. However, I wanted to open the discussion, because PoCs sometimes get "promoted" to official releases, or "borrowed from" by other projects.

Also, as a potential contributor to this codebase, I'd like to follow whatever style guide is in force.

Thanks!

**Comment by gtback** [comment]
Ideally we want to follow PEP8. I typically ignore E501, and occasionally E126, E127, and E128. 

It's on my list to add some automated checking of these. In the past I've used landscape.io, but if anyone knows of alternatives, I'd be happy to consider them.

**Comment by guitarmanvt** [comment]
Travis as amazing. https://travis-ci.com/

**Comment by gtback** [comment]
Yep, Travis is great! I'm hoping to enable it for this repository soon.

**Comment by gtback** [comment]
This was fixed as of 4c2b103ff6727ca134f3d4c36da457c67bc4e795.

Right now it's just checking with pycodestyle. I'd love to incorporate flake8, pylint, or other checkers in the future.
